README.md

```diff
@@ -37,11 +37,12 @@ A repo built on ExpressJS and React with TypeScript
 
 ## Commit Practices
 
-1. Use the _command line_ for commits
+1. Run `yarn format` in the root directory
```

Comment:
what does this do @yuenyauu

Reply:
It formats all the relevant files based on prettier.config, so our codebase's formatting will be consistent